.github/workflows/rust.yml

```diff
@@ -75,3 +73,9 @@ jobs:
       run: (cd key-import; wasm-pack build)
     - name: wasm-pack an example project for export client
       run: (cd key-export; wasm-pack build)
+  check-changelog:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Check changelogs
+      run: ./.github/changelog.sh
```

Comment:
You put the script in the wrong directory, it should be in `.github/` not in `/`

Reply:
Oops